scripts-dev/convert_server_keys.py

```diff
@@ -72,7 +67,7 @@ def rows_v2(server, json):
     valid_until = json["valid_until_ts"]
     key_json = encode_canonical_json(json)
     for key_id in json["verify_keys"]:
-        yield (server, key_id, "-", valid_until, valid_until, db_type(key_json))
+        yield (server, key_id, "-", valid_until, valid_until, db_binary_type(key_json))
```

Comment:
Curious why you didn't just use `memoryview` directly here (and some other places)?

Reply:
I kinda felt like maybe the abstraction was useful. I went back and forth on it a bit :man_shrugging: